subprojects/test-runner/command-line-executor/build.gradle.kts

```diff
@@ -5,11 +5,9 @@ plugins {
 }
 
 publish {
-    artifactId.set("runner-shared")
+    artifactId.set("command-line-executor")
 }
 
 dependencies {
-    compileOnly(gradleApi())
-    api(project(":common:logger"))
     implementation(libs.rxJava)
 }
```

Comment:
Is this module in nature a part of test runner?
It looks more like a common\generic module